src/Pos/Binary/Ssc/GodTossing/Toss.hs

```diff
@@ -0,0 +1,37 @@
+-- | Binary instances for Toss types.
+
+module Pos.Binary.Ssc.GodTossing.Toss
```

Comment:
Long ago we had a discussion about avoiding stuff like `Bi GtTag => bardaq`. The idea was to put types into `Types` module and actual functionality into other modules. Then `Binary` modules can import `Types` and other modules can import `Bi` instances. However, if we put `GodTossing.Toss` and `GodTossing.Core` into `Binary.GodTossing`, then we can't import `Bi` instances from `GodTossing.Core`, because `GodTossing.Core` doesn't know about `Toss`.